### DISCUSSION TITLE
Pipeline sync failure for Zendesk Chat source with S3 avro destination

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34088] [General]
I have created a connection using the following:
Source: Zendesk Chat 
Destination: S3 avro

Sync failure: 
![image](https://github.com/airbytehq/airbyte/assets/106967803/5e4275de-3977-40ed-9d94-923146d01b11)
![image](https://github.com/airbytehq/airbyte/assets/106967803/820f7e09-2a23-4322-a2d7-b60df4bf5227)

Logs are attached below:
[daf368e5_bf24_4faf_befc_95e0488cbe21_job_869_attempt_5_txt.txt](https://github.com/airbytehq/airbyte/files/13886866/daf368e5_bf24_4faf_befc_95e0488cbe21_job_869_attempt_5_txt.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque does this sync work with any other s3 format?

[a comment by an automated account is not shown]
